osu.Game/Screens/Play/HitErrorDisplay/HitErrorDisplay.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics.Containers;
+using osu.Game.Rulesets.Judgements;
+using osu.Game.Rulesets.Objects;
+
+namespace osu.Game.Screens.Play.HitErrorDisplay
+{
+    public abstract class HitErrorDisplay : CompositeDrawable
+    {
+        protected readonly HitWindows HitWindows;
+
+        protected HitErrorDisplay(float overallDifficulty, HitWindows hitWindows)
```

Comment:
i don't think this component should be responsible for applying the OD.  i'd prefer (at very least) the hit windows were taken from the first hitobject in the beatmap instead, where they are already applied.

Reply:
For some reason HitWindows in HitObjects is null